lib/Getopt/Std.pm

```diff
@@ -39,8 +40,8 @@ The C<getopts()> function returns true unless an invalid option was found.
 The C<getopt()> function is similar, but its argument is a string containing
 all switches that take an argument.  If no argument is provided for a switch,
 say, C<y>, the corresponding C<$opt_y> will be set to an undefined value.
-Unspecified switches are silently accepted.  Use of C<getopt()> is not
-recommended.
+Unspecified switches are silently accepted.  Use of C<getopt()> is B<not
```

Comment:
`getopt` is not recommended, make this bold, so that it's not somewhat hidden in the end.